README.rst

```diff
@@ -11,7 +11,7 @@
    `Zope Toolkit project <http://docs.zope.org/zopetoolkit/>`_.
 
 This package represents the core of the Zope Component Architecture.
-Together with the 'zope.interface' package, it provides facilities for
+Together with the ``zope.interface`` package, it provides facilities for
```

Comment:
I'm tempted to actually make this a link to zope.interface (on GitHub?  PyPI?  ReadTheDocs?).